### ISSUE TITLE
Rule proposal: `no-non-exists-builtin-properties`

### ISSUE THREAD

**Issue by fisker** [sindresorhus/eslint-plugin-unicorn#2225]
### Description

I've made two mistakes recently when using `Object.entries()`, they are just typos, but spell check didn't pick them since they are words.

I used `Object.entires()` in one place, and `Object.entire()` .

### Fail

```js
Math.NON_EXISTS_PROPERTY
```

```js
Promise.nonExistsMethod()
```

```js
Function.once()
```

### Pass

```js
Math.PI
```

```js
Promise.race()
```

**Comment by sindresorhus** [comment]
Accepted

**Comment by sindresorhus** [comment]
Rule name should be `no-nonexistent-builtin-properties`

**Comment by sindresorhus** [comment]
Fixed by https://github.com/sindresorhus/eslint-plugin-unicorn/commit/14cc4db51d47d42c8e46ad2fba4b8062d267bbfe
